requirements.txt

```diff
@@ -1,6 +1,33 @@
-matplotlib==3.0.2
-PySide==1.2.4
-# PySide2==5.12.0
-Shiboken==1.2.2
-six==1.12.0
-Markdown==3.2.2
```

Comment:
is this correct? isn't this used by the addon manager?

Reply:
No, I took out Markdown.